### PR TITLE
Updating content-x images

### PR DESCRIPTION
**Pull request by kzollove**
**Summary of changes:**
Accessing Data - 1 Image Moved
Evaluating a data package - 3 images deleted
ORCID - 1 image added
Publishing a data package - 3 images deleted
Tracking a data package - 1 image added
Adding journal citation - 1 image updated
Data Exploration - 2 images updated

### CODE REVIEW

**Comment by clnsmth** [comment]
Thanks @kzollove!

@servilla @sgclarkevt I'll review and merge this PR ASAP.

**Comment by clnsmth** [comment]
@kzollove Just checking these images have been compressed. If not use https://tinypng.com/.

**Comment by kzollove** [comment]
@clnsmth Yes, all images added/updated in this PR went through tinypng

File: templates/resources/evaluating-a-data-package.md [hunk 1]
```diff
@@ -16,8 +16,6 @@ To evaluate a data package via the [EDI Data Portal](https://portal.edirepositor
 
 From the Data Portal, click the **Login** button and enter EDI account credentials.
 
-<img class="screen-shot" src="/static/images/login-portal.png" width="85%">
```
Comment (clnsmth) [inline templates/resources/evaluating-a-data-package.md]: @kzollove Don't remove any images from this page.

File: templates/resources/publishing-a-data-package.md [hunk 2]
```diff
@@ -18,8 +18,6 @@ The process of publishing via the [EDI Data Portal](https://portal.edirepository
 
 From the Data Portal, click the **Login** button and enter EDI account credentials.
 
-<img class="screen-shot" src="/static/images/login-portal.png" width="80%">
```
Comment (clnsmth) [inline templates/resources/publishing-a-data-package.md]: @kzollove Don't remove images from this page.

**Review by clnsmth — CHANGES_REQUESTED** [review]
@kzollove Looks great. One requested change: keep images in the evaluate and publish data pages. Thanks!

File: templates/resources/evaluating-a-data-package.md (hunk 1 above)
Comment (kzollove) [reply templates/resources/evaluating-a-data-package.md]: Images have been restored in the PR

File: templates/resources/publishing-a-data-package.md (hunk 2 above)
Comment (kzollove) [reply templates/resources/publishing-a-data-package.md]: Images have been restored in the PR

**Comment by kzollove** [comment]
Images have been restored to Publishing, Evaluating, and one image in Evaluating has been upgraded

**Pull request merged**